pkg/volume/plugins.go

```diff
@@ -711,7 +711,7 @@ func NewPersistentVolumeRecyclerPodTemplate() *v1.Pod {
 			Containers: []v1.Container{
 				{
 					Name:    "pv-recycler",
-					Image:   "gcr.io/google_containers/busybox",
+					Image:   "busybox:latest",
```

Comment:
Production images **MUST NOT** use unversioned images, since it is impossible to do a staged rollout. Please fix.

Reply:
+1. Please use a versioned image as @tallclair suggested. 
There is also no reason to use the "Always" pull policy that's set by default for ":latest" images.

Reply:
created https://github.com/kubernetes/kubernetes/pull/54192 take care this comment. -Thanks